src/expand/proc_macro.cpp

```diff
@@ -163,15 +163,15 @@ struct ProcMacroInv:
     const ::HIR::ProcMacro& m_proc_macro_desc;
     ::std::ofstream m_dump_file;
 
-#ifdef WIN32
+#ifdef _WIN32
     HANDLE  child_handle;
     HANDLE  child_stdin;
     HANDLE  child_stdout;
 #else
     // POSIX
     pid_t   child_pid;  // Questionably needed
-     int    child_stdin;
-     int    child_stdout;
+    int    child_stdin;
+    int    child_stdout;
```

Comment:
```suggestion
    int     child_stdout;
```

Reply:
You're pointing out the change in indentation? I assume it was visual studio getting oppinionated.

Reply:
Yes, with this change it is aligned again.

Reply:
I reverted the whitespace changes